OpenRA.Mods.Common/Traits/Render/RenderSimple.cs

```diff
@@ -18,6 +18,8 @@ namespace OpenRA.Mods.Common.Traits
 {
 	public class RenderSimpleInfo : RenderSpritesInfo, IRenderActorPreviewSpritesInfo, IQuantizeBodyOrientationInfo, ILegacyEditorRenderInfo, Requires<IBodyOrientationInfo>
 	{
+		public readonly string DefaultSequence = "idle";
```

Comment:
The convention we have set in our other render traits (mainly With*Overlay) is to call this just Sequence.  The "Default" part doesn't add anything useful.